src/wheel/bdist_wheel.py

```diff
@@ -35,6 +36,138 @@
 PY_LIMITED_API_PATTERN = r'cp3\d'
 
 
+def python_tag():
+    return 'py{}'.format(sys.version_info[0])
+
+
+def get_platform(archive_root):
+    """Return our platform name 'win32', 'linux_x86_64'"""
+    # XXX remove distutils dependency
+    result = distutils.util.get_platform()
+    if result.startswith("macosx") and archive_root is not None:
+        result = calculate_macosx_platform_tag(archive_root, result)
+    if result == "linux_x86_64" and sys.maxsize == 2147483647:
+        # pip pull request #3497
+        result = "linux_i686"
+    return result
+
+
+def calculate_macosx_platform_tag(archive_root, platform_tag):
```

Comment:
I think this function should go into the macos specific module.

Reply:
done